tsconfig.json

```diff
@@ -5,11 +5,13 @@
     "removeComments": false,
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "moduleResolution": "Node",
+    "moduleResolution": "node",
     "module": "commonjs",
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowJs": true,
-    "typeRoots": ["../node_modules/@types", "./types/modules"]
-  }
+    "skipLibCheck": true
```

Comment:
This limits TS to check only what's explicitly in our source code, and no external libraries

Reply:
"typeRoots" was constraining the types pulled in to the TS compilation process such that we had to explicitly import them with the triple-slash directives